src/firestore/collection/collection.ts

```diff
@@ -104,12 +104,18 @@ export class AngularFirestoreCollection<T=DocumentData> {
   /**
    * Listen to all documents in the collection and its possible query as an Observable.
    */
-  valueChanges(): Observable<T[]> {
+  valueChanges(idField?: string): Observable<T[]> {
```

Comment:
Maybe something like this for future proofing? Not a huge deal but it could help.

```suggestion
  valueChanges({ idField: string }: ValueChangeOptions?): Observable<T[]> {
```

Reply:
I like that :+1:

Reply:
I second using an options object for future proofing. Great PR otherwise.